### PR TITLE
Add link to Etherpad plugin list in README

### PR DESCRIPTION
**Pull request by ovari**
Link to Etherpad Plugin list

### CODE REVIEW

**Comment by slel** [comment]
Is the change of all the single quotes and double quotes wanted?
Some "smart quotes" mechanism from the text editor or the OS
might be playing this trick. Changing quotes is not needed and
obscures the point of this pull request when looking at the diff.

Also, "Add link to Etherpad plugin list in README" could be a more
informative title for the pull request than "Update README.md",
can you change that?

**Comment by ovari** [comment]
Update title as requested. Can you please merge? Thank you

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ git clone --branch master https://github.com/ether/etherpad-lite.git && cd ether
 ```
 
 ### Manual install
-You'll need git and [node.js](https://nodejs.org) installed (minimum required Node version: **10.13.0**).
+You’ll need git and [node.js](https://nodejs.org) installed (minimum required Node version: **10.13.0**).
```
Comment (JohnMcLear) [inline README.md:28]: please no.  Throughout entire project it's ``'``

Comment (ovari) [reply README.md:28]: Why can’t the project be updated?

Comment (JohnMcLear) [reply README.md:28]: I tend to focus on priorities that matter.  

``'`` is my style and it's the majority of the project.

I don't see value in switching away from ``'``.

**Comment by slel** [comment]
Saw you replaced this by a fresh pull request at #4211. Thanks.

**Pull request closed without merging**